docs/src/main/sphinx/sql/set-time-zone.rst

```diff
@@ -0,0 +1,61 @@
+=============
+SET TIME ZONE
+=============
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    SET TIME ZONE LOCAL
+    SET TIME ZONE expression
+
+Description
+-----------
+
+Sets the current default time zone displacement for the current SQL-session.
+
+If the ```LOCAL`` option is specified, the time zone for the current SQL-session
+will be set to the original time zone of the current SQL-session.
+
+If the ``expression`` option is specified:
+
+- if the type of the ``expression`` is a string, the time zone for the current SQL-session is set to the corresponding region-based time zone ID or the corresponding zone offset.
+- if the type of the ``expression`` is an interval, the time zone for the current SQL-session is set to the corresponding zone offset relative to UTC. It must be in the range of [-14,14] hours.
+
+
+Examples
+--------
+
+Use the default time zone for the current SQL-session::
+
+    SET TIME ZONE LOCAL;
+
+Use a region-based time zone identifier for specifying the time zone::
+
+    SET TIME ZONE 'America/Los_Angeles';
+
+Same outcome can be obtained when the time zone identifier is the output
+of a function call::
+
+    SET TIME ZONE concat_ws('/', 'America', 'Los_Angeles');
+
+Use a zone offset for specifying the time zone::
+
+    SET TIME ZONE '-08:00';
+
+Use an interval literal for specifying the time zone::
+
+    SET TIME ZONE INTERVAL '10' HOUR;
+    SET TIME ZONE INTERVAL -'08:00' HOUR TO MINUTE;
+
+Limitations
+-----------
+
+Setting the default time zone for the session will have no effect if
+the ``sql.forced-session-time-zone`` is already set.
+
+See also
+--------
+
+:doc:`show-session`
```

Comment:
maybe add some context for why show session is useful and what is displayed from this new feature?

Reply:
`show session` is actually not relevant here anymore. 
Thanks for raising this.
Of more relevance here is actually the `current_timestamp()` function. I added the link to it here and removed `show session`.